src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -496,5 +510,27 @@ private async Task<PackageSpec> GetPackageSpecAsync(ISettings settings)
                 }
             };
         }
+
+        /// <inheritdoc/>
+        internal override ValueTask<PackageSpec> GetPackageSpecAsync(CancellationToken ct)
```

Comment:
Same thing as the other comoment applies.

Reply:
Changed to Task<PackageSpec>